[AUTOCUT] OS Distribution Build Failed for sql-2.6.0

***Received Error***: **Error building sql, retry with: ./build.sh manifests/2.6.0/opensearch-2.6.0.yml --component sql --snapshot**.
                      The distribution build for sql has failed.
                      Please see build log at https://build.ci.opensearch.org/job/distribution-build-opensearch/7115/consoleFull

https://github.com/opensearch-project/sql/blob/b82683501de422cd642ffbcad2e0d9f2aa3e6e9a/core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java#L507
Unicode character `...` is rejected:
```
/tmp/tmp__30f1rl/sql/core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java:507: error: unmappable character (0xA6) for encoding US-ASCII
   * return the weekday index for date (1 = Sunday, 2 = Monday, ???, 7 = Saturday).
                                                                  ^
```

--- a/common/src/test/java/org/opensearch/sql/common/grok/BasicTest.java
+++ b/common/src/test/java/org/opensearch/sql/common/grok/BasicTest.java
@@ -113,22 +113,22 @@ public class BasicTest {
     try (FileOutputStream fis = new FileOutputStream(temp);
          BufferedWriter bw = new BufferedWriter(
              new OutputStreamWriter(fis, StandardCharsets.ISO_8859_1))) {
-      bw.write("TEST §");
+      bw.write("TEST #");
     }
 
     GrokCompiler compiler = GrokCompiler.newInstance();
     compiler.register(new FileInputStream(temp), StandardCharsets.ISO_8859_1);
     Grok grok = compiler.compile("%{TEST}");
-    assertEquals("(?<name0>§)", grok.getNamedRegex());
+    assertEquals("(?<name0>#)", grok.getNamedRegex());
   }
 
   @Test
   public void test007_testLoadPatternFromReader() throws IOException, GrokException {
-    Reader reader = new StringReader("TEST €");
+    Reader reader = new StringReader("TEST $");
     GrokCompiler compiler = GrokCompiler.newInstance();
     compiler.register(reader);
     Grok grok = compiler.compile("%{TEST}");
-    assertEquals("(?<name0>€)", grok.getNamedRegex());
+    assertEquals("(?<name0>$)", grok.getNamedRegex());
   }
 
 }

--- a/common/src/test/java/org/opensearch/sql/common/grok/BasicTest.java
+++ b/common/src/test/java/org/opensearch/sql/common/grok/BasicTest.java
@@ -113,22 +113,22 @@ public class BasicTest {
     try (FileOutputStream fis = new FileOutputStream(temp);
          BufferedWriter bw = new BufferedWriter(
              new OutputStreamWriter(fis, StandardCharsets.ISO_8859_1))) {
-      bw.write("TEST #");
+      bw.write("TEST \\u2022");
     }
 
     GrokCompiler compiler = GrokCompiler.newInstance();
     compiler.register(new FileInputStream(temp), StandardCharsets.ISO_8859_1);
     Grok grok = compiler.compile("%{TEST}");
-    assertEquals("(?<name0>#)", grok.getNamedRegex());
+    assertEquals("(?<name0>\\u2022)", grok.getNamedRegex());
   }
 
   @Test
   public void test007_testLoadPatternFromReader() throws IOException, GrokException {
-    Reader reader = new StringReader("TEST $");
+    Reader reader = new StringReader("TEST \\u20AC");
     GrokCompiler compiler = GrokCompiler.newInstance();
     compiler.register(reader);
     Grok grok = compiler.compile("%{TEST}");
-    assertEquals("(?<name0>$)", grok.getNamedRegex());
+    assertEquals("(?<name0>\\u20AC)", grok.getNamedRegex());
   }
 
 }

--- a/core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java
+++ b/core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java
@@ -504,7 +504,7 @@ public class DateTimeFunction {
 
   /**
    * DAYOFWEEK(STRING/DATE/DATETIME/TIME/TIMESTAMP).
-   * return the weekday index for date (1 = Sunday, 2 = Monday, …, 7 = Saturday).
+   * return the weekday index for date (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
    */
   private DefaultFunctionResolver dayOfWeek(FunctionName name) {
     return define(name,

--- a/docs/user/dql/functions.rst
+++ b/docs/user/dql/functions.rst
@@ -1376,7 +1376,7 @@ If an argument of type TIME is provided, the local date is used.
    * - %c
      - Month, numeric (0..12)
    * - %D
-     - Day of the month with English suffix (0th, 1st, 2nd, 3rd, …)
+     - Day of the month with English suffix (0th, 1st, 2nd, 3rd, ...)
    * - %d
      - Day of the month, numeric (00..31)
    * - %e
@@ -1602,7 +1602,7 @@ DAYOFWEEK
 Description
 >>>>>>>>>>>
 
-Usage: dayofweek(date) returns the weekday index for date (1 = Sunday, 2 = Monday, …, 7 = Saturday).
+Usage: dayofweek(date) returns the weekday index for date (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
 
 The `day_of_week` function is also provided as an alias.
 
@@ -2579,7 +2579,7 @@ The functions `weekofyear` and `week_of_year` is also provided as an alias.
    * - Mode
      - First day of week
      - Range
-     - Week 1 is the first week …
+     - Week 1 is the first week ...
    * - 0
      - Sunday
      - 0-53

--- a/docs/user/ppl/functions/datetime.rst
+++ b/docs/user/ppl/functions/datetime.rst
@@ -440,7 +440,7 @@ If an argument of type TIME is provided, the local date is used.
    * - %c
      - Month, numeric (0..12)
    * - %D
-     - Day of the month with English suffix (0th, 1st, 2nd, 3rd, …)
+     - Day of the month with English suffix (0th, 1st, 2nd, 3rd, ...)
    * - %d
      - Day of the month, numeric (00..31)
    * - %e
@@ -694,7 +694,7 @@ DAYOFWEEK
 Description
 >>>>>>>>>>>
 
-Usage: dayofweek(date) returns the weekday index for date (1 = Sunday, 2 = Monday, …, 7 = Saturday).
+Usage: dayofweek(date) returns the weekday index for date (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
 
 Argument type: STRING/DATE/DATETIME/TIMESTAMP
 
@@ -1505,7 +1505,7 @@ Usage: week(date[, mode]) returns the week number for date. If the mode argument
    * - Mode
      - First day of week
      - Range
-     - Week 1 is the first week …
+     - Week 1 is the first week ...
    * - 0
      - Sunday
      - 0-53

--- a/integ-test/src/test/java/org/opensearch/sql/legacy/MethodQueryIT.java
+++ b/integ-test/src/test/java/org/opensearch/sql/legacy/MethodQueryIT.java
@@ -27,8 +27,12 @@ public class MethodQueryIT extends SQLIntegTestCase {
   }
 
   /**
-   * query 搜索就是　，　lucene 原生的搜素方式 注意这个例子中ｖａｌｕｅ可以随便命名 "query" :
-   * {query_string" : {"query" : "address:880 Holmes Lane"}
+   * query
+   * "query" : {
+   *   query_string" : {
+   *     "query" : "address:880 Holmes Lane"
+   *   }
+   * }
    *
    * @throws IOException
    */
@@ -43,8 +47,15 @@ public class MethodQueryIT extends SQLIntegTestCase {
   }
 
   /**
-   * matchQuery 是利用分词结果进行单个字段的搜索． "query" : { "match" : { "address" :
-   * {"query":"880 Holmes Lane", "type" : "boolean" } } }
+   * matchQuery
+   * "query" : {
+   *   "match" : {
+   *     "address" : {
+   *       "query" : "880 Holmes Lane",
+   *       "type" : "boolean"
+   *     }
+   *   }
+   * }
    *
    * @throws IOException
    */
@@ -58,11 +69,45 @@ public class MethodQueryIT extends SQLIntegTestCase {
   }
 
   /**
-   * matchQuery 是利用分词结果进行单个字段的搜索． "query" : { "bool" : { "must" : { "bool" : {
-   * "should" : [ { "constant_score" : { "query" : { "match" : { "address" : {
-   * "query" : "Lane", "type" : "boolean" } } }, "boost" : 100.0 } }, {
-   * "constant_score" : { "query" : { "match" : { "address" : { "query" :
-   * "Street", "type" : "boolean" } } }, "boost" : 0.5 } } ] } } } }
+   * matchQuery
+   * {
+   *   "query": {
+   *     "bool": {
+   *       "must": {
+   *         "bool": {
+   *           "should": [
+   *             {
+   *               "constant_score": {
+   *                 "query": {
+   *                   "match": {
+   *                     "address": {
+   *                       "query": "Lane",
+   *                       "type": "boolean"
+   *                     }
+   *                   }
+   *                 },
+   *                 "boost": 100
+   *               }
+   *             },
+   *             {
+   *               "constant_score": {
+   *                 "query": {
+   *                   "match": {
+   *                     "address": {
+   *                       "query": "Street",
+   *                       "type": "boolean"
+   *                     }
+   *                   }
+   *                 },
+   *                 "boost": 0.5
+   *               }
+   *             }
+   *           ]
+   *         }
+   *       }
+   *     }
+   *   }
+   * }
    *
    * @throws IOException
    */
@@ -102,8 +147,13 @@ public class MethodQueryIT extends SQLIntegTestCase {
   }
 
   /**
-   * wildcardQuery 是用通配符的方式查找某个term 　比如例子中 l*e means leae ltae ....
-   * "wildcard": { "address" : { "wildcard" : "l*e" } }
+   * wildcardQuery
+   * l*e means leae ltae ...
+   * "wildcard": {
+   *   "address" : {
+   *     "wildcard" : "l*e"
+   *   }
+   * }
    *
    * @throws IOException
    */
@@ -117,10 +167,10 @@ public class MethodQueryIT extends SQLIntegTestCase {
   }
 
   /**
-   * matchPhraseQueryTest 短语查询完全匹配．
+   * matchPhraseQuery
    * "address" : {
-   * "query" : "671 Bristol Street",
-   * "type" : "phrase"
+   *   "query" : "671 Bristol Street",
+   *   "type" : "phrase"
    * }
    *
    * @throws IOException

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/Condition.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/Condition.java
@@ -18,7 +18,7 @@ import org.opensearch.sql.legacy.parser.NestedType;
 import org.opensearch.sql.legacy.utils.StringUtils;
 
 /**
- * 过滤条件
+ *
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/Field.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/Field.java
@@ -13,7 +13,7 @@ import org.opensearch.sql.legacy.parser.ChildrenType;
 import org.opensearch.sql.legacy.parser.NestedType;
 
 /**
- * 搜索域
+ *
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/MethodField.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/MethodField.java
@@ -14,7 +14,7 @@ import org.opensearch.sql.legacy.parser.NestedType;
 import org.opensearch.sql.legacy.utils.Util;
 
 /**
- * 搜索域
+ *
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/Order.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/Order.java
@@ -7,7 +7,7 @@
 package org.opensearch.sql.legacy.domain;
 
 /**
- * 排序规则
+ *
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/SearchResult.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/SearchResult.java
@@ -30,7 +30,7 @@ import org.opensearch.sql.legacy.exception.SqlParseException;
 
 public class SearchResult {
     /**
-     * 查询结果
+     *
      */
     private List<Map<String, Object>> results;
 
@@ -82,7 +82,7 @@ public class SearchResult {
     }
 
     /**
-     * 讲es的field域转换为你Object
+     *
      *
      * @param fields
      * @return
@@ -101,7 +101,7 @@ public class SearchResult {
     }
 
     /**
-     * 讲es的field域转换为你Object
+     *
      *
      * @param fields
      * @return

--- a/legacy/src/main/java/org/opensearch/sql/legacy/domain/Select.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/domain/Select.java
@@ -18,7 +18,7 @@ import org.opensearch.sql.legacy.parser.SubQueryExpression;
 
 
 /**
- * 将sql语句转换为select 对象
+ * sql select
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/esdomain/mapping/FieldMappings.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/esdomain/mapping/FieldMappings.java
@@ -27,10 +27,10 @@ import org.opensearch.cluster.metadata.MappingMetadata;
  * 'type': integer
  * },
  * 'state': {
- * 'type': text，
+ * 'type': text
  * }
  * 'name': {
- * 'type': text，
+ * 'type': text
  * 'fields': {
  * 'keyword': {
  * 'type': keyword,

--- a/legacy/src/main/java/org/opensearch/sql/legacy/parser/FieldMaker.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/parser/FieldMaker.java
@@ -41,7 +41,7 @@ import org.opensearch.sql.legacy.utils.SQLFunctions;
 import org.opensearch.sql.legacy.utils.Util;
 
 /**
- * 一些具有参数的一般在 select 函数.或者group by 函数
+ *
  *
  * @author ansj
  */

--- a/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/AggMaker.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/AggMaker.java
@@ -72,7 +72,7 @@ public class AggMaker {
     private Where where;
 
     /**
-     * 分组查的聚合函数
+     *
      *
      * @param field
      * @return
@@ -535,7 +535,7 @@ public class AggMaker {
     }
 
     /**
-     * 按照时间范围分组
+     *
      *
      * @param field
      * @return
@@ -663,7 +663,7 @@ public class AggMaker {
     }
 
     /**
-     * 构建范围查询
+     *
      *
      * @param field
      * @return
@@ -723,7 +723,7 @@ public class AggMaker {
     }
 
     /**
-     * TOPHITS查询
+     * TOPHITS
      *
      * @param field
      * @return

--- a/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/Maker.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/Maker.java
@@ -93,7 +93,7 @@ public abstract class Maker {
     }
 
     /**
-     * 构建过滤条件
+     *
      *
      * @param cond
      * @return

--- a/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/QueryMaker.java
+++ b/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/QueryMaker.java
@@ -19,7 +19,7 @@ import org.opensearch.sql.legacy.exception.SqlParseException;
 public class QueryMaker extends Maker {
 
     /**
-     * 将where条件构建成query
+     *
      *
      * @param where
      * @return
@@ -58,7 +58,7 @@ public class QueryMaker extends Maker {
     }
 
     /**
-     * 增加嵌套插
+     *
      *
      * @param boolQuery
      * @param where

